pkg/fixer/fixes/fixes.go

```diff
@@ -0,0 +1,48 @@
+package fixes
+
+import (
+	"github.com/open-policy-agent/opa/ast"
+	"github.com/open-policy-agent/opa/format"
+)
+
+// NewDefaultFixes returns a list of default fixes that are applied by the fix command.
+// When a new fix is added, it should be added to this list.
+func NewDefaultFixes() []Fix {
+	return []Fix{
+		&Fmt{},
```

Comment:
Not that it matters, but do we even need to include this when the next fix is also calling the formatter?

Reply:
If this were removed, the below case would not be covered:

```
regal $ cat test.rego
package badformatv1

import rego.v1

  allow := true
regal $ go run main.go lint test.rego 
Rule:           opa-fmt                                                 
Description:    File should be formatted with `opa fmt`                 
Category:       style                                                   
Location:       test.rego:1:1                                           
Text:           package badformatv1                                     
Documentation:  https://docs.styra.com/regal/rules/style/opa-fmt        

1 file linted. 1 violation found.
exit status 3
[1]regal $ go run main.go fix test.rego 
No fixes applied.
```

Each fix in the list of NewDefaultFixes has a `Key()`, the key corresponds to the name of a linter error that the fix addresses and triggers the running of the fix.

So if a file doesn't have `use-rego-v1` but is badly formatted, then it would only have a `opa-fmt` violation. And if this were removed, then the file would not be fixed by formatting.

I wonder if it makes sense for `Key()` to be `Name()` to match what we have on rules...

Reply:
Renamed in 8b9e6285662b11afb0a404982b533d6b07350235

Reply:
https://github.com/StyraInc/regal/pull/653/commits/01a38884ebbdcc36a2ea42b01a6c153682a75391

Reply:
I was thinking the default fixes would all be applied, but it's later filtered by actual linter errors is what you're saying? Yeah, that makes total sense then 👍

Reply:
Yeah exactly, this is the default in that it's the 'defaultly enabled list' of fixes to be run when there are linter violations.